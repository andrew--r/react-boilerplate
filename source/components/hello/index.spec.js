import React from 'react';
import renderer from 'react-test-renderer';
import { Hello } from './';

test('should render', () => {
  const component = renderer.create(<Hello />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
