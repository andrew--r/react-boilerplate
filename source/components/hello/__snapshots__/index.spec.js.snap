// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<h1
  className="styles-root-1Q-wE"
>
  Hello, world!
</h1>
`;
